### ISSUE TITLE
Issue with editing and reading source and translation text in Poedit using Control+arrow keys

### ISSUE THREAD

**Issue by tarikhadzirovicofficial** [nvaccess/nvda#17625]
NVDA has a module for Poedit that facilitates the work of translators.
Among other features, there is a functionality that, when pressing Control+Arrow Up or Control+Arrow Down, moves the NVDA focus to the previous or next string, whether it’s the source text or the translation text.
However, there is an issue when attempting to review the text using text navigation commands or when trying to input text. In both cases, NVDA does not announce anything until the focus is moved away from the field and then returned to its original position.

### Steps to reproduce:

1. Open Poedit.
2. Load a translation file (*.po). (You can open it directly, skipping the previous step.)
3. Locate any string in the list of strings.
4. Place the NVDA focus on the source text field or the translation field. Note: If you plan to set the NVDA focus on the translation text field, it must contain some text.

### Actual behavior:

speech viewer enabled
speak command keys on
tab
panel  edit  read only  moves to the next block quote
ctrl+right arrow
to 
ctrl+right arrow
the 
up arrow
moves to the next block quote
ctrl+down arrow
translations list  list  translations list (3057 items)
no next block quote  Translation — Bosnian (Bosnia & Herzegovina): nema sljedećeg citata  438 of 3057
right arrow
right arrow
right arrow
up arrow
up arrow
up arrow
ctrl+right arrow
ctrl+left arrow
ctrl+right arrow
ctrl+left arrow
NVDA+2
speak typed characters on
ctrl+left arrow
ctrl+right arrow
NVDA+2
speak typed characters off
tab
Needs work  check box  not checked
shift+tab
panel  edit  read only  no next block quote
ctrl+right arrow
quote
ctrl+right arrow
kraj retka
ctrl+left arrow
quote
ctrl+left arrow
block 
up arrow
no next block quote
NVDA+4
speak command keys off
speech viewer disabled

### Expected behavior:

NVDA should read the text without any issues using the text navigation commands.

### NVDA logs, crash dumps and other attachments:

I have not attached at the moment (I can if necessary)

#### NVDA installed/portable/running from source:

Installed.

#### NVDA version:

2024.4.

#### Windows version:

Windows 11 23H2 (AMD64) build 22631.4602

#### Name and version of other software in use when reproducing the issue:

Poedit 3.5.2.

#### Does the issue still occur after restarting your computer?

Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No.

#### If NVDA add-ons are disabled, is your problem still occurring?

Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes.

**Comment by gerald-hartig** [comment]
This is a Poedit bug. Please raise the issue with the authors.

**Comment by michaelDCurran** [comment]
Poedit is firing a focus event on the translations list but not actually physically moving the system focus. Thus keys still go to the text field.
